gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -1865,3 +1970,47 @@ nnstreamer_converter_set_custom_property_desc (const char *name,
       varargs);
   va_end (varargs);
 }
+
+/**
+ * @brief Registers a callback for tensor_converter custom condition
+ * @return 0 if success. -ERRNO if error.
+ */
+int
+nnstreamer_converter_custom_register (const gchar * name,
+    tensor_converter_custom func, void *data)
+{
+  converter_custom_cb_s *ptr;
+
+  g_return_val_if_fail (name && strlen (name), -EINVAL);
+  g_return_val_if_fail (func, -EINVAL);
+
+  if (!(ptr = g_try_new0 (converter_custom_cb_s, 1)))
+    return -ENOMEM;
+
+  ptr->func = func;
+  ptr->data = data;
+
+  if (register_subplugin (NNS_CUSTOM_CONVERTER, name, ptr) == TRUE)
+    return 0;
+
```

Comment:
if failed to register custom, release allocated ptr

Reply:
Updated.